packages/apps/galaxpay/assets/onload-expression.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+  window._galaxyHashcard = function (card) {
+    return new Promise((resolve, reject) => {
+      // https://docs.galaxpay.com.br/tokenizacao-cartao-js
+      const token = window._galaxPayPublicToken;
+
+      // !window._galaxPaySandbox // false = sandbox, true = production
+      const isLive = !window._galaxPaySandbox || true;
```

Comment:
```suggestion
      const isLive = !window._galaxPaySandbox;
```

` || true` era a gambi porque não tava achando o bug? 😄